test/factories/cluster_group.rb

```diff
@@ -0,0 +1,42 @@
+FactoryBot.define do
+  # gets a study object, defaulting to the first user found.  Auto-appends a unique number to the name, and a note
+  # in the description of the study, to aid in test DB uniqueness and cleanup efforts
+  factory :cluster_group do
+    cluster_type { '2d' }
```

Comment:
Given `{name: :z, type: 'coordinates'}` below, it seems `cluster_type` should be parameterized.

Reply:
Thanks for correcting my misunderstanding -- this already is parameter, and '2d' is the default.  What you have here seems ideal.